rapids-cmake/cpm/versions.json

```diff
@@ -79,8 +78,8 @@
     },
     "rmm": {
       "version": "${rapids-cmake-version}",
-      "git_url": "https://github.com/rapidsai/rmm.git",
-      "git_tag": "branch-${version}"
+      "git_url": "https://github.com/vyasr/rmm.git",
+      "git_tag": "feat/rapids_logger_library"
```

Comment:
This will be removed before merging. I made this change for the various repo tests.

Reply:
All the generate-pins test failures are caused by this right now. It's trying to check out a branch that I don't have on my fork of rmm.